app/src/main/java/com/taufiqrahman/ratingreviews/GradientActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.taufiqrahman.ratingreviews;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.util.Pair;
+
+import com.taufiqrahman.reviewratings.BarLabels;
+import com.taufiqrahman.reviewratings.RatingReviews;
+
+import java.util.Random;
+
+public class GradientActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_gradient);
+
+
+        RatingReviews ratingReviews = (RatingReviews) findViewById(R.id.rating_reviews);
+
+        Pair colors[] = new Pair[]{
+            new Pair<>(Color.parseColor("#0e9d58"), Color.parseColor("#1e88e5")),
+            new Pair<>(Color.parseColor("#bfd047"), Color.parseColor("#5c6bc0")),
+            new Pair<>(Color.parseColor("#ffc105"), Color.parseColor("#d81b60")),
+            new Pair<>(Color.parseColor("#ef7e14"), Color.parseColor("#8bc34a")),
+            new Pair<>(Color.parseColor("#d36259"), Color.parseColor("#ea80fc"))
+        };
+
+        int minValue = 30;
```

Comment:
added a min value for showcasing because, in some cases, the random number would be so small that you couldn't see the gradient.

Reply:
Understandable. Might add a comment on that right there. Or just use fix ratings rather than random `int`s. Your choice.